pypeit_files/gemini_gmos_gn_ham_r400_885.pypeit

```diff
@@ -4,13 +4,13 @@
 # User-defined execution parameters
 [rdx]
     spectrograph = gemini_gmos_north_ham
-    #slitspatnum = MSC01:1079 
+    slitspatnum = MSC01:1079 
 [calibrations]
     [[wavelengths]]
         lamps=['OH_GMOS']
         method = 'holy-grail'
-        rms_threshold = 1.,0.3,0.4,0.5
-        sigdetect = 5.,5.,6.,6.
+        rms_threshold = 1.,0.3,0.4
```

Comment:
https://github.com/pypeit/PypeIt/pull/1628 deprecates using a list for `rms_threshold`

In https://github.com/pypeit/PypeIt-development-suite/pull/272, 
I have simply set this to 0.4

Please do the same here